examples/bcic_iv_2a_cropped.py

```diff
@@ -69,37 +66,24 @@
 assert len(raw_train.ch_names) == 22
 # lets convert to millvolt for numerical stability of next operations
 raw_train = mne_apply(lambda a: a * 1e6, raw_train)
```

Comment:
could we use raw. apply_function here instead of a custom mne_apply?

Reply:
@agramfort mne_ext submodule with function mne_apply is removed in PR #82. Function calls to mne_apply are replaced by raw.apply_function()